src/main/java/frc/team3388/robot/SystemFactory.java

```diff
@@ -21,4 +24,14 @@ public ExampleSystem createExampleSystem() {
 
         return new ExampleSystem(motor);
     }
+
+    public HopperSystem createHopperSystem() {
+        SpeedController motor = new SpeedControllers()
+                .add(new WPI_TalonSRX(RobotMap.HOPPER_MOTOR))
+                .build();
+
+        ColorSensorV3 proximitySensor = new ColorSensorV3(I2C.Port.kOnboard);
```

Comment:
Actually.. I made a mistake.
The sensor is connected via the Talon SRX. So it needs to look like:

```java
WPI_TalonSRX talon = ...;
...
ColorSensorV3 sensor = new ColorSensorV3(talon);
...
```

Reply:
My bad